test/prog.c

```diff
@@ -1,6 +1,8 @@
 /* Copyright (c) 2017 - 2018 LiteSpeed Technologies Inc.  See LICENSE. */
+//To use getaddrinfo in windows
```

Comment:
Is this comment misplaced?  Note that we use the C-style comments `/* like this */`.

Reply:
I simply forgot to remove it. It did describe some code I deleted.